src/mdframe/reader.py

```diff
@@ -57,8 +62,9 @@ def main():
                         default="toml")
     args = parser.parse_args()
     config = Config(
-        Path(args.directory),
-        args.metadata_ext,
+        data_dir_path = Path(args.directory),
```

Comment:
Good usage of kwargs. One nitpick, when passing kwargs you should skip the space around `=`, so e.g. `data_dir_path=Path(args.directory)`. Elsewhere, you include whitespace around operators.

Reply:
taken care of :thumbsup: